app/src/main/java/org/oppia/android/app/player/exploration/BottomSheetOptionsMenu.kt

```diff
@@ -0,0 +1,38 @@
+package org.oppia.android.app.player.exploration
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import org.oppia.android.R
+import org.oppia.android.databinding.BottomSheetOptionsMenuFragmentBinding
+
+/** Bottom sheet fragment for displaying options menu */
+class BottomSheetOptionsMenu : BottomSheetDialogFragment() {
```

Comment:
`InjectableDialogFragment ` should we have a `InjectableBottomSheetDialogFragment` too which should be used here. For reference check any dialog fragment.

Reply:
Added InjectableBottomSheetDialogFragment